Makefile

```diff
@@ -3,6 +3,12 @@ VERBOSE ?= false
 RETRY_ATTEMPTS ?= 10
 RETRY_SECONDS ?= 30
 
+ifneq ($(strip $(AGENT_DIR)),)
+    AGENT_DIR_COMPOSED = $(ROOT_DIR)/$(AGENT_DIR)
```

Comment:
It's different because of when it's executed on the pipeline, root_dir and cur_dir can be different, one is the directory from the repository where this is being triggered (ROOT_DIR) and the other is the directory of the action (CUR_DIR) since now the agent_dir can be overridden by a custom agent_dir in the e2e folder of the repo where the action is used, that's why there are two different ones. It's not possible to converge.